test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -5249,92 +5249,118 @@ private void testModifiedFolderModifiedDocumentsQuery(String query,
         ImmutableMap.of("documentum.modifiedDocumentsQuery", query),
         startPaths(folder),
         new Checkpoint(FEB_1970, DOCUMENT.pad("aaa")),
-        makeExpectedDocIds(folder, expected),
+        expected,
         new Checkpoint(MAR_1970, DOCUMENT.pad("bbb")));
   }
 
   @Test
   public void testModifiedDocumentsQuery() throws Exception {
     String folder = "/FFF1/FFF2";
     String query = "SELECT i_chronicle_id, r_object_id, object_name, "
-        +"r_modify_date, "
+        + "r_modify_date, "
+        + "DATETOSTRING_LOCAL(r_modify_date, ''yyyy-mm-dd hh:mi:ss'') "
+        + "AS r_modify_date_str FROM dm_sysobject "
+        + "WHERE FOLDER(''/FFF1'',descend) "
+        + "AND ((r_modify_date = DATE(''{0}'',''yyyy-mm-dd hh:mi:ss'') "
+        + "AND r_object_id > ''{1}'') "
+        + "OR (r_modify_date > DATE(''{0}'',''yyyy-mm-dd hh:mi:ss''))) "
+        + "UNION "
+        + "SELECT i_chronicle_id, r_object_id, object_name, r_modify_date, "
         + "DATETOSTRING_LOCAL(r_modify_date, ''yyyy-mm-dd hh:mi:ss'') "
-        +"AS r_modify_date_str "+"FROM dm_sysobject "
-        +"WHERE FOLDER(''/FFF1'',descend) "
-        +"AND ((r_modify_date = DATE(''{0}'',''yyyy-mm-dd hh:mi:ss'') "
-        +"AND r_object_id > ''{1}'') "
-        +"OR (r_modify_date > DATE(''{0}'',''yyyy-mm-dd hh:mi:ss''))) "
-        +"UNION "
-        +"SELECT i_chronicle_id, r_object_id, object_name, r_modify_date, "
-        +"DATETOSTRING_LOCAL(r_modify_date, ''yyyy-mm-dd hh:mi:ss'') "
-        +"AS r_modify_date_str "
-        +"FROM dm_sysobject "
-        +"WHERE r_object_id = ''" + DOCUMENT.pad("aaa") + "'' "
-        +"ORDER BY 4 ASC,2 ASC ENABLE(RETURN_TOP 500)";
+        + "AS r_modify_date_str "
+        + "FROM dm_sysobject "
```

Comment:
Maybe fewer line breaks? This isn't even consistent with AS ... FROM ... above. Starting lines with SQL keywords is nice, but I think it's better to bend that rule when appending quoted values, rather than have + "'' " on a line by itself.

If you really want the keywords, I'd rather have the WHERE clause on one line as you had, though I do prefer the one + per line.